src/lib.rs

```diff
@@ -1,4 +1,24 @@
 //! A small library for generating badge svg images
+//! ## Example
```

Comment:
Please add an empty line here for better formatting :)

Reply:
resolved https://github.com/relaxdays/badges/pull/1/commits/2d2e05aea376380d79f0fa8164b2802931677142